pkg/apps/registry/deployconfig/strategy.go

```diff
@@ -152,6 +152,10 @@ func (statusStrategy) PrepareForUpdate(ctx apirequest.Context, obj, old runtime.
 	oldDc := old.(*deployapi.DeploymentConfig)
 	newDc.Spec = oldDc.Spec
 	newDc.Labels = oldDc.Labels
+
+	if newDc.Status.LatestVersion != oldDc.Status.LatestVersion {
+		newDc.Generation = oldDc.Generation + 1
```

Comment:
@tnozicka that is spec vs. status... @kargakis thoughts?

Reply:
I don't think we need this here. The hack inside the spec strategy linked by @tnozicka is in place because `oc deploy` was bumping status.latestVersion prior to splitting spec/status in DCs. Nothing (ie. the DC controller today) that updates /status should touch latestVersion.

Reply:
Btw, either 3.7 or 3.8 we should be able to get rid of the hack altogether since we have `oc rollout latest` in the client. Depends on when we added the latter.